src/DynamicObj/DynamicObj.fs

```diff
@@ -21,11 +21,8 @@ type DynamicObj() =
     /// Gets property value
     member this.TryGetValue name = 
```

Comment:
Does this fix #25 ? If so, please add a regression test

Reply:
Nope, not related